hook.py

```diff
@@ -9,4 +10,11 @@
 async def initialize(app, services):
     Popen(['haproxy', '-q', '-f', 'plugins/ssl/templates/haproxy.conf'])
     logging.debug('Serving at https://127.0.0.1:443')
+    config = open('plugins/ssl/templates/haproxy.conf', 'r').read()
+    path = config[config.index('crt') + 4:config.index('.pem') + 4]
```

Comment:
this still feels too brittle. what if we name the ssl_cert.pem to insecure_certificate.pem (which is something that Vagrant does, sort of). Then, we can check if that certificate is in the conf/ directory and warn based off of that. it's not perfect but i think it's simple enough.